### PR TITLE
Added myself as Reddit Advocate

### PR DESCRIPTION
**Pull request by jjasghar**
- I created the subreddit, and have moderator ability.

### CODE REVIEW

**Comment by adamleff** [comment]
:+1:

**Comment by lamont-granquist** [comment]
👍

**Comment by btm** [comment]
RFC020 specifies a majority of existing advocates, but isn't clear about how we handle new areas of advocating. That said, JJ is already an advoate, this is just a new area. We got +1's from three existing advocates which is about half, and everyone else in the irc meeting was in favor including some maintainers.

That'll do. thanks @jjasghar.

File: ADVOCATES.md [hunk 1]
```diff
@@ -63,4 +63,5 @@ More information can be found in the [Chef Community Guidelines](https://github.
 * Vacant - Submit a PR to become a Chef Community Advocate
 
 #### Chef_Opscode Reddit ([https://www.reddit.com/r/chef_opscode/](https://www.reddit.com/r/chef_opscode/))
```
Comment (lamont-granquist) [inline ADVOCATES.md]: ^ that should probably be renamed to something non-opscode, but r/chef obviously doesn't work and i have no better idea...

Comment (jjasghar) [inline ADVOCATES.md]: Yep, I'd love to, but I don't know if it's possible, i chose `/r/chef_opscode` three years ago because `/r/chef` private.

**Pull request merged**